### PR TITLE
replace marker light by new marker light tagging

### LINKED ISSUE #179: New tagging of marker lights
The tagging scheme for German marker lights ("Kennlicht") has been changed:

    railway:signal:*:marker_light=yes -> railway:signal:*:states=*;DE-ESO:kennlicht

see e.g. http://wiki.openstreetmap.org/wiki/DE:OpenRailwayMap/Tagging_in_Germany#Ks-Hauptsignale

Add a validator rule throwing an error for occurences of the old scheme and change the German JOSM preset.

### PR DESCRIPTION
**Pull request by Nakaner**
This fixes #179 

I replaced the old marker light tagging by the new one and translated some strings at Hp semaphore signals.

I removed marker lights at Vr semaphore signals because they are impossible (they would violate Eisenbahn-Signalordnung).

### CODE REVIEW

**Comment by DerDakon** [comment]
You add 2 empty lines at line 1 and 201.

File: josm-presets/de-signals-eso.xml [hunk 1]
```diff
@@ -195,6 +197,14 @@ See http://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 						delete_if_empty="true" />
 					<key key="railway:signal:main:states"
```
Comment (DerDakon) [inline josm-presets/de-signals-eso.xml]: Here you have still a <key> for the states, for the other signals you removed that.

File: josm-presets/de-signals-eso.xml [hunk 2]
```diff
@@ -195,6 +197,14 @@ See http://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
...
 						value="DE-ESO:hp0;DE-ESO:ks1" />
+					<combo key="railway:signal:main:states" default="DE-ESO:hp0;DE-ESO:ks1;DE-ESO:ks2"
```
Comment (DerDakon) [inline josm-presets/de-signals-eso.xml]: The default is not in the list of values.

File: josm-presets/de-signals-eso.xml [hunk 3]
```diff
@@ -707,6 +703,14 @@ See http://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 						value="light" />
 					<key key="railway:signal:distant:states"
```
Comment (DerDakon) [inline josm-presets/de-signals-eso.xml]: Here is also a duplicate <key> for the states.

File: josm-presets/de-signals-eso.xml [hunk 4]
```diff
@@ -1528,11 +1515,6 @@ See http://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 						display_values="Formsignal (Sh 0/Sh 1),Lichtsignal (Hp 0/Sh 1),Zugdeckungssignal (Hp 0/Kennlicht)"
```
Comment (DerDakon) [inline josm-presets/de-signals-eso.xml]: The values line here is wrong, the last ';' should be ','. Not your fault, but if you fix the marker light stuff, then please fix that, too.

**Comment by Nakaner** [comment]
@DerDakon I have fixed your issues.

**Pull request merged**